tar/ccache/ccache_write.c

```diff
@@ -83,6 +87,10 @@ callback_write_rec(void * cookie, uint8_t * s, size_t slen, void * rec)
 	if (ccr->age > MAXAGE)
 		goto done;
 
+	/* Don't write an entry if it has negative mtime. */
```

Comment:
Actually, we might as well have all of the "Don't write an entry if" checks before the assertions.